Analysis/backgrounds_hfake.tex

```diff
@@ -60,9 +63,17 @@
   }
   \label{fig:hadronTFactor}
 \end{figure}
+\PH{Can labels be consistent with the captions proxy....also can you
+  make this clear that this is data. Given that where are the error
+  bars (minor point). Also the caption doesn't specify the origin of hte uncertainty.} 
 
 
-Under the assumption that the $R_{h}$ stays constant regardless of whether the event has a high-\pt\ jet or \met, the hadron proxy sample is weighted by $R_{h}$ to determine the number of events due to misidentified hadrons in the signal region.
+Under the assumption that the $R_{h}$ stays constant regardless of
+whether the event has a high-\pt\ jet or \met, the hadron proxy sample
+is weighted by $R_{h}$ to determine the number of events due to
+misidentified hadrons in the signal region.
+\PH{How is signal overlap dealt with in this region and the photon
+  veto. I don't recall a MET cut, will ask during talk.}
```

Comment:
clarified this section some. the photon ID criteria are orthogonal to the signal region... the giant control region table should help.